src/datepicker/types.js

```diff
@@ -163,6 +166,7 @@ export type CalendarPropsT = {
 
 export type HeaderPropsT = CalendarPropsT & {
   date: Date,
+  order: number,
```

Comment:
can you describe the `order` prop's purpose? I have a hard time reasoning about what it does by reading the type definition

Reply:
to put it more clearly, can this prop be named something more descriptive? maybe index?

Reply:
This prop [already existed](https://github.com/uber-web/baseui/blob/6389c9b3e3044ba983487e1e22189b05a3f4158c/src/datepicker/calendar.js#L172) but wasn't defined in the props for some reason. I dont mind changing the name but would rather issue it in a separate diff